taccsite_cms/templatetags/blog_cats_organized.py

```diff
@@ -0,0 +1,24 @@
+from django import template
+from django.conf import settings
+
+TACC_BLOG_CATEGORY_ORDER = settings.TACC_BLOG_CATEGORY_ORDER
+register = template.Library()
```

Comment:
This has been done previously in Core-CMS by passing settings through template to the templatetag (as in https://github.com/TACC/Core-CMS/pull/594), but I'll review this alternative after the holidays. I'm not opposed; I just wanna think on it.

Reply:
This is a sensible way to access the setting, and better than how I would have done it.

@taoteg, I learned something. If you already knew it, just ignore me. The `SETTINGS_EXPORT` is not required for explicitly using a setting in the context of a template; a templatetag can import one instead. 🙂

@R-Tomas-Gonzalez, Do you think (outside this PR) you could change `TACC_BLOG_CUSTOM_MEDIA_POST_CATEGORY` to be used this same way?

Reply:
Will do.